rs/orchestrator/registry_replicator/src/lib.rs

```diff
@@ -295,17 +334,15 @@ impl RegistryReplicator {
         }
 
         info!(
-            self.logger,
+            logger,
             "Finished local store initialization at registry version: {}", registry_version
         );
     }
 
     /// Initializes the registry local store asynchronously and returns a future that
     /// continuously polls for registry updates.
-    pub async fn start_polling(
+    pub fn start_polling(
         &self,
-        nns_urls: Vec<Url>,
-        nns_pub_key: Option<ThresholdSigPublicKey>,
         cancellation_token: CancellationToken,
     ) -> Result<impl Future<Output = ()> + use<>, Error> {
```

Comment:
nit: does this work?
```suggestion
    pub async fn start_polling( &self, cancellation_token: CancellationToken
) -> Result<(), Error> {
```